exercises/farewell-workflow/practice/Web/Web.http

```diff
@@ -0,0 +1,13 @@
+@Web_HostAddress = http://localhost:5125
+
+### Get Spanish Greeting

```

Comment:
I'm not familiar with `.http` files in dotnet applications but Github's syntax highlighting is suggesting that it doesn't like comments/headers formatted this way... could be wrong and it's fine, just seems worth a head check.